src/System.Net.Http/tests/StressTests/HttpStress/ChecksumHelpers.cs

```diff
@@ -57,17 +57,17 @@ public static ulong update_crc(ulong crc, byte[] buf, int len)
             return c;
         }
 
-        public static ulong update_crc(ulong crc, string text, Encoding encoding = null)
+        public static ulong update_crc(ulong crc, string text, Encoding? encoding = null)
```

Comment:
How did this compile without warnings before?  Or there were warnings and we're missing warn-as-error?